runbotics/runbotics-orchestrator-ui/src/views/process/ProcessBuildView/Modeler/ActionListPanel/ActionSearch.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from 'react';
+import { Autocomplete, TextField, AutocompleteProps } from '@mui/material';
+import { translate } from 'src/hooks/useTranslations';
+import { Item } from '../ListGroup';
+
+export type SearchBarProps = Omit<AutocompleteProps<Item, false, false, false>, 'renderInput'> & {};
+
+const ActionSearch = ({ ...other }: SearchBarProps) => {
+    const [inputValue, setInputValue] = useState('');
+    const [highlightedValue, setHighlightedValue] = useState<Item>();
+    const [value, setValue] = useState<Item>(null);
```

Comment:
imo it would be better to have a single state with 2 fields in it (isHighlighted, label). Cause now you have 2 sync states, thus there can be some unwanted app states.